components/FilterSelection/index.tsx

```diff
@@ -162,7 +162,7 @@ const topics = [
 const publishers = [
   { heading: "All publishers" },
   {
-    heading: "Department for Business, Energy & Industrial Strategy",
+    heading: "The Department for Business, Energy & Industrial Strategy",
```

Comment:
Gareth made me take all the "The " prefixes out of the api the other day, we should probably be consistent.

Wider point, we've got a getPublishers fetch around here somewhere, can we just fetch these?

Reply:
I think this got pulled in when we was making some changes.
there's a ticket in this sprint for it, hooking these filter selections up to the api is next on my todo list